ocaml/xenopsd/lib/xenops_server.ml

```diff
@@ -2122,6 +2123,8 @@ let rec perform_atomic ~progress_callback ?subtask:_ ?result (op : atomic)
   | VM_delay (id, t) ->
       debug "VM %s: waiting for %.2f before next VM action" id t ;
       Thread.delay t
+  | VM_softreboot id ->
```

Comment:
Add a debug line here, like all the cases above.

Reply:
Given the CLI works which uses this function to translate between the database and CLI I would say so, it copies from the crash behaviour

Reply:
Wrong comment?

Reply:
2 above